Microsoft.Toolkit.Uwp.UI.Controls/Expander/ExpanderAutomationPeer.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+
+namespace Microsoft.Toolkit.Uwp.UI.Automation.Peers
+{
+    /// <summary>
+    /// Defines a framework element automation peer for the <see cref="Expander"/> control.
+    /// </summary>
+    public class ExpanderAutomationPeer : FrameworkElementAutomationPeer, IToggleProvider
```

Comment:
@jamesmcroft I missed we should have used the `IExpandCollapseProvider` in addition or instead of maybe? Thoughts?

Reply:
Let me take a look, I don't see why not though. Would you want this to be inclusive of `IToggleProvider` or in favor of?

Reply:
@jamesmcroft good question. I don't know if it's a problem to provide multiple interfaces, probably best to include both if possible? I know some other code will probably look for one over the other, so I imagine having both would allow it to be used by the most other code parts looking for certain types of peers. @chingucoding @ranjeshj any thoughts?

Would it make sense to add unit tests like in #3544 as well?

Reply:
> I don't know if it's a problem to provide multiple interfaces, probably best to include both if possible?

I would argue that having both interfaces only makes sense if you expose both patterns in `GetAutomationPatternCore`. So at least implement what you are providing as part of `GetAutomationPatternCore`. For what it's worth, the WinUI expander control only specifies the ExpandCollapsePattern, so I am inclined to say that that pattern/interface is sufficient.

> Would it make sense to add unit tests like in #3544 as well?

Yes, I think testing that peer and the exposed functions of the peer should be done.

Reply:
You can have multiple patterns supported for sure. [ExpandCollapse ](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern) pattern made sense for Expander.

Reply:
@jamesmcroft if you've got some cycles still to continue helping with this, mind submitting a follow-on PR with the tweaks? Thanks!

Reply:
@michael-hawker I'll get another PR open for these changes 😄